packages/manager/modules/models/billing/Bill.class.js

```diff
@@ -0,0 +1,36 @@
+export default class Bill {
+  constructor({
+    billId,
+    category,
+    date,
+    debt,
+    pdfBetaUrl,
+    pdfUrl,
+    priceWithTax,
+    priceWithoutTax,
+    url,
+  }) {
+    Object.assign(this, {
+      billId,
+      category,
+      date,
+      pdfBetaUrl,
+      pdfUrl,
+      priceWithTax,
+      priceWithoutTax,
+      url,
+    });
+
+    this.debt = debt || {
+      status: 'PAID',
+      dueAmount: {
+        value: 0,
+        text: priceWithTax.text.replace(/[0-9.]*/, '0.00'),
```

Comment:
Be aware that some countries format prices with a comma (or a white space, that would break too) :
``` '3,100.5'.replace(/[0-9.]*/, '0.00') ```
``` "0.00,100.5" ```

Reply:
Nice catch ! I've updated the regex